<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CADDigitizeDockWidgetBase</class>
 <widget class="QDockWidget" name="CADDigitizeDockWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1234</width>
    <height>219</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="windowTitle">
   <string>CADDigitize</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QLabel" name="ToolInfo">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>781</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="ToolCMD">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>781</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>45</y>
      <width>781</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="ShapeInfo">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>34</y>
      <width>361</width>
      <height>131</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="chk_refresh">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>171</y>
      <width>361</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>11</y>
      <width>361</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Shape:</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>781</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="shapeOptionsLayout"/>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
